main/src/com/google/refine/importing/ImportingUtilities.java

```diff
@@ -539,13 +542,15 @@ static public long saveStreamToFile(InputStream stream, File file, SavingUpdate
                 }
             }
             return length;
+        } catch (ZipException e) {
+            throw new IOException("Compression format not supported, " + e.getMessage());
```

Comment:
Are we sure that all `ZipException`s are related to the fact that the compression format is not supported? Intuitively there could be other cases, where the zip archive is corrupted, for instance?

Reply:
Would have to do some more research here, but that makes sense. Perhaps we can catch Exception and forward along the error message rather than specifying the compression format is not supported.

Reply:
Looking into this - I think this current way makes sense. Looking at microsoft docs it is throws "when the file or stream is not a valid zip file."

While that could mean a corrupted file I think that can be bucketed into compression format not supported since it is not a valid compression in the first place. Any other non ZipException will be throws as IOException and still propagated up to the UI but without the additional text added for ZipException.